osi_sensorview.proto

```diff
@@ -344,3 +349,15 @@ message UltrasonicSensorView
     //
     optional UltrasonicSensorViewConfiguration view_configuration = 1;
 }
+
+//
+// \brief Definition of the microphone sensor view.
+//
+// Ultrasonic specific sensor view data.
+//
+message MicrophoneSensorView
+{
+    // Microphone view configuration valid at the time the data was created.
+    //
+    optional MicrophoneSensorViewConfiguration view_configuration = 1;
```

Comment:
What would the return data (in a virtual environment) be for a Microphone "sensor"
For example CameraSensroView have "image_data".

Reply:
The return data will be the frequency spectrum the microphone has detected.